docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/03-Digital-Ocean-(manual).md

```diff
@@ -45,8 +51,6 @@ ssh root@37.139.15.166
 ssh root@37.139.15.166 -i /Users/demo/.ssh/id_rsa_do
```

Comment:
Show this code snippet with a placeholder first:

```bash
ssh root@__IP_ADDRESS__ -i /Users/.ssh/id_rsa_do
```

 and then add another one saying smth like "and this is what it looks like with a real-world value"